utils/callData/autoRenewalCalls.ts

```diff
@@ -29,10 +30,13 @@ function enableRenewal(
   };
 }
 
-function disableRenewal(encodedDomain: string): Call[] {
+function disableRenewal(
+  autoRenewalContract: string,
+  encodedDomain: string
+): Call[] {
   return [
     {
-      contractAddress: process.env.NEXT_PUBLIC_RENEWAL_CONTRACT as string,
```

Comment:
So we need to delete the env variable on Vercel also ?

Reply:
No as I updated the constants to use the env variable